python/revng/model/_yaml.py

```diff
@@ -0,0 +1,28 @@
+import yaml
+
+
+class YamlLoader(yaml.Loader):
+    pass
```

Comment:
I guess we can drop this

Reply:
We could, I left it there so that it's possible to globally customize the YamlLoader as well as YamlDumper by applying modifications to the base class, without touching generated files. As of now there's no such usage, that's true, so your call if you want me to remove it.